source/go-perun/tutorial/getting-started.rst

```diff
@@ -32,8 +32,8 @@ Restart your shell and check the installation by running::
 Ganache CLI
 -----------
 
-For the purpose of this tutorial we will use `ganache-cli <https://github.com/trufflesuite/ganache-cli>`_ for providing us with a local Ethereum blockchain for testing our application locally.
-Install ganache-cli by following `the instructions on the web page <https://github.com/trufflesuite/ganache-cli#installation>`_.
+For the purpose of this tutorial we will use `ganache-cli <https://github.com/trufflesuite/ganache>`_ for providing us with a local Ethereum blockchain for testing our application locally.
+Install ganache-cli by following `the instructions on the web page <https://github.com/trufflesuite/ganache#getting-started>`_.
```

Comment:
I noticed the CI pipeline fails due to error in this link check. 

It is because, checking for anchors in GitHub Markdown files failed and we had an exception for this specific link in [`conf.py`](https://github.com/cryptphil/perun-doc/blob/6f3a391638dfec1f6a7e2a1f2012f69e533b4cdb/source/conf.py#L181). Since the link has changed from `ganache-cli` to `ganache`, the exception is not valid anymore.

Can you please the update the updated link in the configuration file mentioned above ? @cryptphil